### PR TITLE
[WIP] Update to use latest Pipelines Beta code

### PR DESCRIPTION
**Pull request by a-roberts**
https://github.com/tektoncd/dashboard/issues/1063

Tested I can get and create resources fine (including import, PipelineRuns, reruns), and viewing Trigger resources

### CODE REVIEW

**Comment by a-roberts** [comment]
Adding @AlanGreene and @vdemeester as reviewers to keep an eye on this one 😸 (the code could be cleaner, but this works - see description for testing and obvious todo item)

File: src/api/index.js [hunk 1]
```diff
@@ -88,11 +93,22 @@ export function getResourcesAPI(
...
+  version
+) {
+  let versionToUse = 'v1beta1';
+  if (version) {
+    versionToUse = version;
+  }
+  const resourceAPI = getResourcesAPI(
+    { group: 'tekton.dev', type, name, namespace },
+    queryParams,
+    versionToUse
```
Comment (AlanGreene) [inline src/api/index.js]:
```suggestion
export function getTektonAPI(
  type,
  { name = '', namespace, version = 'v1beta1' } = {},
  queryParams
) {
  const resourceAPI = getResourcesAPI(
    { group: 'tekton.dev', type, version, name, namespace },
    queryParams
```

we can set the default version in the first param and avoid the extra juggling here
then just pass it down to `getResourcesAPI` which would remain unchanged (as-is on master)

**Comment by a-roberts** [comment]
Revisited this today as there's an rc cut for PIpelines beta, everything working ok apart from the version detection which is in https://github.com/tektoncd/dashboard/pull/1104

File: Gopkg.toml [hunk 2]
```diff
@@ -6,10 +6,10 @@ required = [
 
 [[constraint]]
   name = "github.com/tektoncd/pipeline"
-  revision = "558b37edfffa1d8a33428fb9bd50a8ab425e3e83"
+  revision = "60d76bd5526e815cdbf5639ff3fa5b9eb09ce321"
```
Comment (vdemeester) [inline Gopkg.toml]: why not using version now ? `version = "v0.11.0-rc1"`

Comment (a-roberts) [reply Gopkg.toml]: Fixed 🤗

**Comment by a-roberts** [comment]
Looking pretty nice now I reckon and tests are green, for awareness (this shouldn't be news to @AlanGreene and @vdemeester from our working group calls, lemme know if it is) the plan is to do a 0.5.3 first (our last "non-beta" Dashboard release) and then a 0.6.0 for beta and that's when this pretty small but important PR can go in

**Comment by anneqm** [comment]
@a-roberts I prefer the 0.5.3 first so we have options for an update for Kabanero before the move to beta.

**Comment by a-roberts** [comment]
Modified this to use a new tekton-beta branch, my plan is to, once the 0.5.3 release is done today, get this published somewhere for folks to try (potentially @afrittoli in the robocat cluster), I'll rebase if necessary and pick up the latest published RC too

I'm unavailable after this Thursday but I'd like to...
1) have the tekton-beta branch done by then to the best of our ability and published
2) have a working release by then that folks can play with (referencing builds of the above ^^)
3) then once we know that's all great and tested and beta goes live, we can merge this into master and update our docs

**Comment by a-roberts** [comment]
I'm gonna merge this into pipelines-beta so we have a stream we can deliver any beta specific fixes into, and then get folks to review this for master when we're ready (this is instead of it sitting at my fork)

**Pull request merged**